src/test/java/ir/sahab/dockercomposer/UserIdTest.java

```diff
@@ -0,0 +1,23 @@
+package ir.sahab.dockercomposer;
+
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class UserIdTest {
```

Comment:
I think this is not good scenario for what are you intend to test. You can actually create container and run "id -u" in it and check if result matches expected user ID. I think that would be more straightforward and clear.